apps/web/components/SearchSection/SearchSection.tsx

```diff
@@ -0,0 +1,225 @@
+import React, { ReactNode, useEffect, useState } from 'react'
+import { useWindowSize } from 'react-use'
+import dynamic from 'next/dynamic'
+import {
+  Text,
+  GridContainer,
+  GridRow,
+  GridColumn,
+  Box,
+  Stack,
+  Inline,
+  Tag,
+  Link,
+} from '@island.is/island-ui/core'
+import { theme } from '@island.is/island-ui/theme'
+import { Locale } from '@island.is/shared/types'
+import { SearchInput } from '@island.is/web/components'
+import { LinkType, useLinkResolver } from '@island.is/web/hooks/useLinkResolver'
+import { GetFrontpageQuery } from '@island.is/web/graphql/schema'
+
+import * as styles from './SearchSection.css'
+
+const DefaultIllustration = dynamic(() => import('./Illustration'), {
+  ssr: false,
+})
+
+type SearchSectionProps = {
+  page: GetFrontpageQuery['getFrontpage']
+  headingId: string
+  activeLocale: Locale
+  quickContentLabel: string
+  placeholder: string
+  browserVideoUnsupported: string
+}
+
+export const SearchSection = ({
+  headingId,
+  activeLocale,
+  quickContentLabel = '',
+  placeholder = '',
+  page,
+}: SearchSectionProps) => {
+  const { width } = useWindowSize()
+  const [isMobile, setIsMobile] = useState<boolean | null>(null)
+  const { linkResolver } = useLinkResolver()
+
+  const {
+    featured,
+    heading,
+    image,
+    imageAlternativeText,
+    imageMobile,
+    videos,
+    videosMobile,
+  } = page
+
+  useEffect(() => {
+    setIsMobile(width < theme.breakpoints.md)
+  }, [width])
+
+  const defaultIllustration = (
+    <DefaultIllustration className={styles.defaultIllustration} />
+  )
+
+  const mobileImage = image?.url ? (
+    <Image src={image?.url} alt={imageAlternativeText} spacing={isMobile} />
+  ) : (
+    defaultIllustration
+  )
+
+  const desktopImage = imageMobile?.url ? (
+    <Image
+      src={imageMobile?.url}
```

Comment:
Desktop image has `imageMobile?.url` and spacing `isMobile`.

Is this intended?

Then above

```mobileImage = image?.url ...```